core-services/ci-secret-generator/_config.yaml

```diff
@@ -241,3 +242,25 @@
   params:
     cluster:
     - app.ci
+- fields:
+  - cmd: /usr/bin/oc_create_kubeconfig.sh /tmp/build-farm-credentials $(cluster) $(service_account)
+      ci
+    name: sa.multi-arch-builder-controller.multi01.token.txt
```

Comment:
I do not see the reason of needing this for pull/push credentials to compose `docker.json`.